### ISSUE TITLE
None metrics are passed to the LearningRateWithMetricsWrapper.step method even though it does not support none. 

### ISSUE THREAD

**Issue by murphp15** [allenai/allennlp#1259]
As stated in the title. 
The LearningRateWithMetricsWrapper.step method does not support None as a value for the metrics object. 
However when it is being trained the method _train_epoch() in trainer.py always passes in None as the metric value.  
This leads to a configuration issue. This seems like a bug to me unless I am missing something. 
Line of code from trainer.py is below. 

`self._update_learning_rate(None, batch_num_total=batch_num_total)
`

**Comment by DeNeutoy** [comment]
Going to be fixed by https://github.com/allenai/allennlp/pull/1255 I think
